src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1503,6 +1511,84 @@ private function registerPropertyInfoConfiguration(array $config, ContainerBuild
         }
     }
 
+    private function registerLockConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
+    {
+        $loader->load('lock.xml');
+
+        $container->getDefinition('lock.store.flock')->replaceArgument(0, sys_get_temp_dir());
```

Comment:
This inlines the tempdir use at compile time for reuse at build time, making the dumped container  not-independent from the build env.

Reply:
good catch